### ISSUE TITLE
Build fails with TypeScript 4.2

### ISSUE THREAD

**Issue by aikoven** [deeplay-io/nice-grpc#2]
```
node_modules/nice-grpc/lib/client/Client.d.ts:6:16 - error TS2322: Type 'KnownKeys<Service>' is not assignable to type 'string | number | symbol'.
  Type 'unknown' is not assignable to type 'string | number | symbol'.
    Type 'unknown' is not assignable to type 'symbol'.
      Type 'KnownKeys<Service>' is not assignable to type 'symbol'.
        Type 'unknown' is not assignable to type 'symbol'.

6     [Method in KnownKeys<Service>]: ClientMethod<Service[Method], CallOptionsExt>;
                 ~~~~~~~~~~~~~~~~~~
```

**Comment by OmkarKirpan** [comment]
I am facing the same issue

**Comment by OmkarKirpan** [comment]
Build fails with TypeScript 4.2
So, I am currently using typescript:4.0.3 exact package to build.

**Comment by aikoven** [comment]
Fixed in 📦 nice-grpc@0.1.3
